pom.xml

```diff
@@ -124,6 +155,12 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-web</artifactId>
             <scope>test</scope>
+            <exclusions>
+                <exclusion>
+                    <groupId>ch.qos.logback</groupId>
+                    <artifactId>logback-classic</artifactId>
+                </exclusion>
+            </exclusions>
```

Comment:
Why we need exclude this module?

Reply:
I had some problems related to logback when I added the ReportServiceTest to this repo. This exclusion was part of Tristan update to this test. @Tristan-WorkGH can you explain ?

Reply:
Add a comment to say it's because there already `slf4j-simple` (an slf4j implementation) in the classpath.